Sources/SwiftSyntaxBuilderGeneration/Node.swift

```diff
@@ -28,18 +28,6 @@ class Node {
   let elementsSeparatedByNewline: Bool
   let collectionElement: String
 
-  var baseType: String {
-    if self.baseKind == "SyntaxCollection" {
-      return "Syntax"
-    } else {
-      return kindToType(kind: baseKind)
-    }
-  }
-
-  var collectionElementType: String {
-    return kindToType(kind: collectionElement)
-  }
```

Comment:
These are replaced with `baseType: SyntaxBuildableType` and `collectionElementType: SyntaxBuildableType`, i.e. an API break to be aware of.